packages/lab/stories/date-picker/date-picker.stories.tsx

```diff
@@ -466,6 +469,199 @@ RangeControlled.parameters = {
   },
 };
 
+export const SingleGridPanel: StoryFn<
```

Comment:
In this story (same on site), if I set number of columns to 2, number of months in grid to 2. Open calendar, use left chevron to navigate left, number of months in the calendar increases

Reply:
was an issue of duplicate keys, introduced by biome change last night,